packages/building-rollup/modern-and-legacy-config.js

```diff
@@ -16,6 +16,8 @@ const getWorkboxConfig = require('@open-wc/building-utils/get-workbox-config');
 const production = !process.env.ROLLUP_WATCH;
 const prefix = '[owc-building-rollup]';
 
+const DEFAULT_DEDUPE_MODULES = ['lit-html', 'lit-element'];
```

Comment:
Unfortuantely, we need to list all deep imports here as well :( see: https://github.com/rollup/plugins/issues/82

We could scan the file system for all modules perhaps?